pkg/webreg/webreg.go

```diff
@@ -1104,6 +1104,11 @@ Example of a <code>ci-operator</code> configuration that overrides a workflow fi
 
 {{ yamlSyntax (index . "configExample2") }}
 
+The configuration can also override a workflow field with a <a href="#step">full literal step</a>
+(not only a reference to a shared step):
+
+{{ yamlSyntax (index . "configExample3") }}
```

Comment:
Can you move this and the example 2 to be beneath "’s field has priority (i.e. the value from the ci-operator configuration is used). "?

Reply:
Sure, makes sense.